docs/source/platform-features/control/authentication-commands.rst

```diff
@@ -462,3 +476,37 @@ A request can be deleted using the following command:
 .. code-block::
 
     vctl auth remote delete <USER_ID>
+
+
+Dynamic RPC Method Authorization
+================================
+RPC method authorizations are the capabilities used to limit access to specific exported RPC methods on an agent.
+While the capability field is used to define which exported RPC methods the agent can access, the rpc_method_authorization
+field describes which capabilities will authorize a remote agent to access it's exported RPC methods.
+
+.. note::
+    While this field can be modified manually, it is best practice to use the interface.
+    When the agent starts up, the AuthService will automatically query it for all current allowed rpc capabilities on each method.
+
+The format for rpc_method_authorizations is as follows:
+
+.. code-block:: json
+
+    rpc_method_authorizations: {
+        "RPC_exported_method_1": [
+            "authorized_capability_1",
+            "authorized_capability_2"
+            ],
+        "RPC_exported_method_2": [
+            "authorized_capability_3",
+            ]
+    }
+
+To dynamically modify an RPC method's authorization, use:
+
+.. code-block:: console
+
+    vctl auth rpc add <agent_id.method> <authorized capability 1> <authorized capability 2> ...
```

Comment:
Please provide a real example here...just the template is not enough for this complex of a thing

Reply:
Added example for adding and removing an authorization, and the consequences of each.